app/views/followships/create.js.erb

```diff
@@ -0,0 +1,7 @@
+
+$('.btnn').bind('ajax:success',function() {
+
+  $(this).closest('div').find('a').fadeOut();
+  $(this).closest('div').append("<span>Followed</span>")
+
```

Comment:
Please update this code. As I have told you before. use classes name

Reply:
Done